README.md

```diff
@@ -1,59 +1,85 @@
-# Security Guard
+# Guard
 
-## Security - How Guard help secure Knative services
-The core component of Guard is the [`guard-gate`](https:pkg/guard-gate) which can be used to extend Knative Queue Proxy. [`guard-gate`](https:pkg/guard-gate) monitor and potentialy block requests and/or responses to Knative services based on a per-service security configuration (Micro-rules). 
+## Why Do We Need Guard?
 
-Micro-rules enable a fine grain filtering performed against each value delivered to/from the service.
-By using micro-rules, [`guard-gate`](https:pkg/guard-gate) can identify the delivery of exploits targeting vulnerabilities embedded as part of the service or its dependencies.
+User containers deployed by Knative may include vulnerabilities, may be misconfigured and may include malicious code. The source of such vulnerabilities, misconfigurations or malicious code may be the DevOps team, a dependency or a hacker that has successfully penetrated any of the support systems (image repository, ci/cd, Knative, Kube, the DevOps team development systems etc. etc. etc.) or one of the services used by the user container. Any such security issue may enable an attacker to use the user container for other purposes than its original intention (e.g. steal data, attack others, spread, contact a C&C, Crypto mining, etc.)
```

Comment:
I would use a bullet list for the different types of vulnerabilities. Just so that its easy to see and improve (like adding more)